pkg/test/framework/components/echo/cmd/echogen/README.md

```diff
@@ -0,0 +1,37 @@
+# echogen
+
+`echogen` is a util for generating Kubernetes manifests from echo configurations.
+
+###Installation:
+
+```bash
+go install istio.io/pkg/test/framework/components/echo/echogen
+```
+
+### Usage:
+
+```text
+echogen [opts] config.yaml
+```
+
+The config file is YAML containing a list of [echo.Config](https://github.com/istio/istio/blob/master/pkg/test/framework/components/echo/config.go#L52) objects:
+
+```yaml
+- Service: a
+  Namespace: echo
+- Service: headless
+  Namespace: echo
+  Headless: true
+```
+
```

Comment:
Could we also document a little bit how to send requests manually with the echo app? Some example commands for `grpcui` or `grpcurl` should be really helpful.

Reply:
Added a brief tutorial :)